exercises/concept/elyses-destructured-enchantments/enchantments.spec.js

```diff
@@ -1,17 +1,5 @@
 // @ts-check
```

Comment:
Not related to this file: The stub file `enchantments.js` currently contains a long comment below `// @ts-check`. Please remove that one. We don't show it anymore at this point in the track.

Reply:
👍 [`3df28d0` (#1274)](https://github.com/exercism/javascript/pull/1274/commits/3df28d08ec59c71337c2593c3860b4f4a4d4c132)